nvflare/edge/examples/jobs/xor_mobile/app/custom/edge_executorch_controller.py

```diff
@@ -89,22 +93,20 @@ def control_flow(self, abort_signal: Signal, fl_ctx: FLContext) -> None:
                     return
 
                 self.log_info(fl_ctx, f"Round {self.current_round} started.")
-                fl_ctx.set_prop(AppConstants.CURRENT_ROUND, self.current_round, private=True, sticky=True)
+                fl_ctx.set_prop(
+                    AppConstants.CURRENT_ROUND,
+                    self.current_round,
+                    private=True,
+                    sticky=True,
+                )
                 self.fire_event(AppEventType.ROUND_STARTED, fl_ctx)
 
                 # Create task and send global model to clients
                 encoded_buffer = self._export_current_model()
 
                 # Compose shareable
                 task_data = Shareable()
-                model = {
-                    "weights": encoded_buffer,
-                    "format": "base64_encoded",
```

Comment:
Can we just use "base64"?

Reply:
sounds good, let me make this one clearer